htdocs/societe/list.php

```diff
@@ -2157,6 +2179,24 @@
 				$totalarray['nbfield']++;
 			}
 		}
+		// Note public
+		if (!empty($arrayfields['s.note_public']['checked'])) {
+			print '<td class="sensiblehtmlcontent center">';
+			print dolPrintHTML($obj->note_public);
+			print '</td>';
+			if (!$i) {
+				$totalarray['nbfield']++;
+			}
+		}
+		// Note private
+		if (!empty($arrayfields['s.note_private']['checked'])) {
+			print '<td class="flat maxwidth250imp">';
+			print dolPrintHTML($obj->note_private);
```

Comment:
Because content of notes can be very large html content, i recommand to do
dolPrintHtml(dolGetFirstLineOfText(note, 5))
Can you try this ?

Reply:
Done